packages/utils/src/date-util/fecha.ts

```
@@ -0,0 +1,343 @@
+  }
+})
+
+fecha.format = (dateObj, mask, i18nSettings) => {
+  const i18n = i18nSettings || fecha.i18n
+
+  if (typeof dateObj === 'number') {
+    dateObj = new Date(dateObj)
+  }
+
+  if (!isDate(dateObj) || isNaN(dateObj.getTime())) {
+    throw new Error('Invalid Date in fecha.format')
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)